src/activation.jl

```diff
@@ -17,9 +12,9 @@ const RealOrFloatType = Union{Real, FloatType}
 Classic [sigmoid](https://en.wikipedia.org/wiki/Sigmoid_function) activation
 function.
 """
-function σ(x::RealOrFloatType)
+function σ(x::Real)
```

Comment:
What about removing these type restrictions? So that  people can overload broadcast without change the model to have looovectorization kick in. @AStupidBear

Reply:
Probably although I don't know why that isn't possible without it?

Reply:
Yes, it's better to remove the `Real` restriction so those functions can be used later in other packages.

Reply:
Currently all these functions are defined for Reals so I think it makes sense to keep that type annotation, but always happy to get rid of the type annotations that can make other types work more generically.